src/components/ChatLog.js

```diff
@@ -0,0 +1,47 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import {DateTime} from 'luxon';
+
+
+const ChatLog = (props) => {
+    const chatComponents = entries => (entries.map(entry => {
```

Comment:
Taking `entries` as a param is a good pattern to try, though in this case you could just use `props.entries` directly and avoid creating a function. Good practice for more advanced needs, though!